package.json

```
@@ -23,32 +23,35 @@
   },
   "scripts": {
     "cleanup": "shx rm -rf dist/",
-    "build": "yarn cleanup && tsc -p config&& rollup -c config/rollup.config.js",
+    "build": "yarn cleanup && tsc -p config && rollup -c config/rollup.config.js && rollup -c config/rollup.config.js --environment NODE_ENV:production",
+    "postbuild": "yarn size && yarn typingsheader",
     "test": "jest -c ./config/jest.config.js --runInBand",
     "test:watch": "yarn test --watch",
     "test:coverage": "yarn test --coverage",
     "test:ci": "yarn test --ci",
-    "validate-js": "tsc -p ./config && tsc -p ./scripts",
-    "verify": "yarn validate-js && yarn style && yarn test:ci",
-    "commit": "git-cz",
+    "size": "node scripts/file-size ./dist/bundles/meilisearch.umd.min.js ./dist/bundles/meilisearch.esm.min.js",
     "style": "yarn format:fix && yarn lint",
     "style:fix": "yarn format:fix && yarn lint:fix",
     "format": "prettier \"**/*.{ts,tsx,js,jsx,css,scss,sass,less,md}\"",
     "format:fix": "yarn format --write",
-    "lint": "tslint --project tsconfig.json --format codeFrame",
+    "lint": "tslint --project config/tsconfig.json -c config/tslint.json --format codeFrame",
     "lint:fix": "yarn lint --fix",
-    "prerelease": "yarn build",
-    "release": "standard-version",
-    "postrelease": "node scripts/copy && node scripts/build && yarn release:github && yarn release:npm",
-    "release:github": "git push --no-verify --follow-tags origin master",
-    "release:npm": "yarn publish dist",
-    "release:preflight": "yarn pack ./dist --dry-run",
-    "size": "node scripts/file-size ./dist/bundles/index.umd.min.js ./dist/bundles/index.esm.min.js"
+    "typingsheader": "node scripts/build.js"
```

Why this one has not a coma?
```suggestion
    "typingsheader": "node scripts/build.js",
```

---

Because it is a `json` file and you can not add a trailing comma in `json` files.